### ISSUE TITLE
Error on youtube8m feature extraction: Op type not registered 'DecodeJpeg'

### ISSUE THREAD

**Issue by thanasisfitsios** [google/mediapipe#2569]
**System information** 
- OS Platform and Distribution: Mac OS BigSur 11.6 
- Bazel version: 4.2.1
- Mediapipe version: 0.8.7.1
- Solution: Youtube8m
- Programming Language and version: Python 3.7.11
- Tried with both Tensorflow 1.14 and 2.6, had the same issue

Hi! I have been trying to extract video features with youtube8m following the instructions [here](https://google.github.io/mediapipe/solutions/youtube_8m). In Step 5, I can build without problem, but I get errors when I try to execute: 
`GLOG_logtostderr=1 bazel-bin/mediapipe/examples/desktop/youtube8m/extract_yt8m_features \
  --calculator_graph_config_file=mediapipe/graphs/youtube8m/feature_extraction.pbtxt \
  --input_side_packets=input_sequence_example=/tmp/mediapipe/metadata.pb  \
  --output_side_packets=output_sequence_example=/tmp/mediapipe/features.pb` 

I get a very long list of messages of this type (For shortness I copy some examples):
 `2021-09-21 11:36:33.927794: E external/org_tensorflow/tensorflow/core/framework/op_kernel.cc:1623] OpKernel ('op: "MutableHashTableOfTensors" device_type: "CPU" constraint { name: "key_dtype" allowed_values { list { type: DT_STRING } } } constraint { name: "value_dtype" allowed_values { list { type: DT_INT64 } } }') for unknown op: MutableHashTableOfTensors
2021-09-21 11:36:33.927933: E external/org_tensorflow/tensorflow/core/framework/op_kernel.cc:1623] OpKernel ('op: "RestoreSlice" device_type: "CPU"') for unknown op: RestoreSlice
2021-09-21 11:36:33.927977: E external/org_tensorflow/tensorflow/core/framework/op_kernel.cc:1623] OpKernel ('op: "BarrierInsertMany" device_type: "CPU" constraint { name: "T" allowed_values { list { type: DT_UINT64 } } }') for unknown op: BarrierInsertMany`

In this long list of unknown ops, one of them is also: `2021-09-21 11:35:01.050847: E external/org_tensorflow/tensorflow/core/framework/op_kernel.cc:1623] OpKernel ('op: "DecodeJpeg" device_type: "CPU"') for unknown op: DecodeJpeg`

After the above messages, execution ends with this output: 
`I20210921 11:36:34.549583 233873408 tensorflow_session_from_frozen_graph_calculator.cc:159] Loaded frozen model in: 1044351 microseconds.
W20210921 11:36:34.584870 320499200 opencv_video_decoder_calculator.cc:230] Not all the frames are decoded (total frames: 1125 vs decoded frames: 0).
E20210921 11:36:34.624042 320499200 extract_yt8m_features.cc:132] Failed to run the graph: CalculatorGraph::Run() failed in Run: 
Calculator::Open() for node "TensorFlowSessionFromFrozenGraphCalculator_1" failed: ; Create failed: Not found: Op type not registered 'DecodeJpeg' in binary running on largoF.local. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.`

I'm not sure if this only a problem with DecodeJpeg, or if I would get the same error with any other 'unknown' op.

I appreciate any help!

**Comment by thanasisfitsios** [comment]
Update: I finally was able to solve the problem. What worked for me was changing the `TENSORFLOW_GIT_COMMIT` in the WORKSPACE file to the latest stable release of tensorflow.

**Comment by sgowroji** [comment]
Glad you resolved and thank you for sharing your fix, Which is definitely help full to the community.

**Comment by versusniubility** [comment]
@thanasisfitsios I'm sorry I've new to tensorflow, May I know what do you mean by changing the TENSORFLOW_GIT_COMMIT in the WORKSPACE file to the latest stable release of tensorflow.? What is TENSORFLOW_GIT_COMMIT and what's the definition of  WORKSPACE file, or what did you do, cause you mentioned that tf 2.6 also doesn't work, I have the same issue.

**Comment by versusniubility** [comment]
@thanasisfitsios Thank you for your reply, though I have got some other issues, I've got a main question, how did you find the sha256, is that provided on GitHub or your calculate yourself, could you give a brief introduction? tkx!

**Comment by thanasisfitsios** [comment]
@versusniubility For the sha256, I ran it with the previous "wrong" value and the error message I got actually gave me the correct value! (It looked something like: "The SHA key you gave is wrong, the correct should be .... etc.")

With the patches, I'm sorry but I don't remember the details of what I did. I think I just manually searched some files that were giving errors and changed something there (e.g. eigen3 to eigen).
